eval_metrics.py

```diff
@@ -0,0 +1,62 @@
+from abstractmodel import AbstractLabelModel
+from dumb_model import DumbModel
+import fire
+from typing import TypedDict
+import pandas as pd
+import common_utils
+
+
+class ParsedDataset(TypedDict):
+    train: list[pd.DataFrame]
+    test: list[pd.DataFrame]
+    split: list[pd.DataFrame]
+
+
+def eval_model(model: AbstractLabelModel, parsed_dataset: ParsedDataset):
+    result = {"train": {}, "test": {}, "eval": {}}
+    for split_name, split in parsed_dataset.items():
+        count_cos_sim = []
+        for subgroup in split:
+            true_label = subgroup["category"][0]
```

Comment:
Should we use `category` as `true_label`? For example for vec2text our `generated_label` will be a longer text, and therefore cos similarity might achieve poor score.

Reply:
I agree with @basiekjusz

Reply:
I'll make an issue about that. We'll talk about it during our meeting